src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
```

Comment:
u haven't plugged fonts, just imported it

Reply:
Fix it

Reply:
Set fonts to the `html` tag selector and make inputs inherit it